_unittest/test_03_Materials.py

```diff
@@ -527,3 +528,14 @@ def test_16_import_materials_from_workbench(self):
             self.testapp2.materials.material_keys["84zn_12ag_4au_imp"].thermal_expansion_coefficient.thermalmodifier
             == "pwl($TM_84Zn_12Ag_4Au_imp_thermal_expansion_coefficient, Temp)"
         )
+
+    def test_17_json_missing_rgb(self, caplog: pytest.LogCaptureFixture):
```

Comment:
I don't want to ask too much but could you mock the JSON content instead of adding data to the repo ?
I'd like to avoid adding files when the file content isn't really worth it. Here the goal is to trigger the warning so having only on material should be enough and the associated dictionary shouldn't be too huge.

Reply:
Done!